lesson2/average.py

```diff
@@ -0,0 +1,10 @@
+def avar(lstx):
+    sumx = 0
+    for i in range(len(lstx)):
```

Comment:
c++ for cycle style. Use python style.
```suggestion
    for i in numbers:
```